### PR TITLE
Feature/itsdocument backwards compatible

### PR DESCRIPTION
**Pull request by mtrupkin**

### CODE REVIEW

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+package tds.itemrenderer.data;
+
+// Represents the raw ITS XML data.
+
+public class ITSDocumentXml extends ITSDocument
+{
+    private long itemKey;
+    private String layout;
+
+    public String getFormat() {
```
Comment (millerts) [inline TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java]: Doesn't this override the ITS Document getFormat()? Can you use the `@Override` where appropriate?  I know it doesn't enforce anything but makes it easier to understand the flow at a quick glance.

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    public void setLayout(String value) {
+        layout = value;
+    }
+
+    public String getLayout() {
+        return layout;
+    }
+
+    public String getResponseType() {
```
Comment (millerts) [inline TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java]: I assume this is reverting back to the previous implementation?  Where this is always forced to be null? Same with the other methods you're overriding with returning null?

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+package tds.itemrenderer.data;
+
+import java.util.List;
+
+/**
+ * @author mskhan
+ *
+ */
+public class IrisITSDocument extends ITSDocument
+{
```
Comment (millerts) [inline TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java]: Can we put the curly bracket on the same line as the class?

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    stimulusKey = itsDocument.getStimulusKey();
+
+    this.setBankKey(itsDocument.getBankKey());
+    this.setBaseUri(itsDocument.getBaseUri());
+    this.setFormat(itsDocument.getFormat());
+    this.setType(itsDocument.getType());
+    this.setRealPath(realPath);
+  }
+
+  public String getFormat() {
```
Comment (millerts) [inline TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java]: I think all methods here should have the `Override` annotation

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java (hunk 1 above)
Comment (mtrupkin) [reply TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java]: Adding the override annotation.

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java (hunk 2 above)
Comment (mtrupkin) [reply TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/ITSDocumentXml.java]: Correct.  Adding the previous behavior of ITSDocumentXML back into the code.

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java (hunk 3 above)
Comment (mtrupkin) [reply TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java]: Ok.

File: TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java (hunk 4 above)
Comment (mtrupkin) [reply TDS.ItemRenderer/src/main/java/tds/itemrenderer/data/IrisITSDocument.java]: Adding the override annotation.

**Pull request merged**